app/templates/profile.hbs

```diff
@@ -9,10 +9,10 @@
       <span class="profile-title">My Profile</span>
       <img class="settings-button" src="https://png.icons8.com/gear/p1em/30/ffffff">
     </div>
-    <div class="search-div">
-        {{search-bar placeholder="Search Profile"}}
-        <img src="https://png.icons8.com/search-filled/ios7/39/00B5AF">
-   </div>
+        {{search-bar
+          placeholder="Search Profile"
```

Comment:
I will eventually get around to adding a variable to the profile controller to populate the value placeholder.